app/tests/spec/lib/router.js

```diff
@@ -5,16 +5,13 @@
 define(function (require, exports, module) {
   'use strict';
```

Comment:
@philbooth - we have two sets of tests for expected routes, one [here in the server tests](https://github.com/mozilla/fxa-content-server/blob/1fa05b074324c1bfbe03423ff33ea03b77ebc169/tests/server/routes.js#L30) and another [here in the functional tests](https://github.com/mozilla/fxa-content-server/blob/1fa05b074324c1bfbe03423ff33ea03b77ebc169/tests/functional/pages.js#L12). Adding a 3rd set might not provide much overall value.